### PR TITLE
[FEATURE] [MER-3598] Migrate Revision History to workspaces/course_author

### PR DESCRIPTION
**Pull request by manelli**
Migrate the `Revision History` to a new liveview under the workspaces/course_author scope.

https://github.com/user-attachments/assets/0300e948-cf1d-472a-93c3-cf24c32ca8d2

See: https://eliterate.atlassian.net/browse/MER-3598

### CODE REVIEW

File: lib/oli_web/live/workspaces/course_author/history_live.ex [hunk 1]
```diff
@@ -0,0 +1,733 @@
...
+        </div>
+      <% end %>
+    </div>
+
+    <script
+      id="keep-alive"
+      type="text/javascript"
+      src={Routes.static_path(OliWeb.Endpoint, "/js/keepalive.js")}
+    >
+    </script>
```
Comment (gastonabella) [inline lib/oli_web/live/workspaces/course_author/history_live.ex:359]: Do you think this code block is necessary? It is already included in `live.html.heex` which is included in the second line of the module with `use OliWeb, :live_view`. I might be missing something, but I'm concerned that we might be showing flash messages twice in this liveview.

**Review by gastonabella — COMMENTED** [review]
Looks great!

I noticed from the attached recording that when visiting this new authoring view, it still selects `Student` in the workspace sidebar on the left. Is this expected or should we have `Course Author` selected instead?

**Comment by manelli** [comment]
> Looks great!
> 
> I noticed from the attached recording that when visiting this new authoring view, it still selects `Student` in the workspace sidebar on the left. Is this expected or should we have `Course Author` selected instead?

This was because I branched out from an outdated master branch that had that problem with the sidebar, I rebased and it is now showing the correct tab.

image

File: lib/oli_web/live/workspaces/course_author/history_live.ex (hunk 1 above)
Comment (manelli) [reply lib/oli_web/live/workspaces/course_author/history_live.ex]: Nice catch, yeah sorry I was testing something and forgot to remove it.
Removed in 7e65a8789babca228d4cd9e467004a37997d41e8

**Review by gastonabella — APPROVED** [review]

**Pull request merged**